### DISCUSSION TITLE
Canadian URL fails to register

### DISCUSSION THREAD

**Discussion by bingwersen** [audiamus/BookLibConnect#62] [Q&A] [answered]
Hi, when I try to register with a Canadian URL, I can get through the logic screen, but then get a message about a non functioning address.

Message:
"We're Sorry.  The Web Address you entered in not a functioning page on out site."

**Comment by audiamus** [comment]
That's Audible misleading you. It basically tells you the contrary: you are in. However, you have to copy the URL of that page from your browser back into the Book Lib Connect New Profile dialog, just as the instructions tell you.

**Comment by bingwersen** [comment, marked as answer]
Brilliant, I'll give it a try.  That you so much for your patience.
